ckanext/dcatapchharvest/tests/fixtures/dataset.json

```diff
@@ -104,7 +104,19 @@
       "documentation": [
         "https://example.com/documentation-resource-1",
         "https://example.com/documentation-resource-2"
-      ]
+      ],
+      "rights": "Creative Commons CC Zero License (cc-zero)",
+      "license": "NonCommercialAllowed-CommercialAllowed-ReferenceNotRequired"
+    },
+    {
+      "id": "e2c50e70-67ad-4f86-bb1b-3f93867eadda",
+      "package_id": "4b6fe9ca-dc77-4cec-92a4-55d6624a5bd6",
+      "documentation": [
+        "https://example.com/documentation-resource-1",
+        "https://example.com/documentation-resource-2"
+      ],
+      "rights": "http://dcat-ap.ch/vocabulary/licenses/terms_by",
+      "license": "NonCommercialAllowed-CommercialAllowed-ReferenceRequired"
```

Comment:
Could we also include a case where the license is a uri, as we will be mapping it in the future?